frontend/src/views/admin/CategoryEdit.jsx

```diff
@@ -21,6 +25,20 @@ const CategoryEdit = () => {
         pageButtons.push(<button key={i} className={`btn btn-sm ${pageNo === i ? "btn-active" : ""}`} onClick={() => setPageNo(i)}>{i + 1}</button>)
     }
 
+    const handleDelete = (event, category) => {
+        axiosInstance.delete(
+            `categories/${category.slug}`,
+            {
+                headers: {
+                    'Authorization': `Bearer ${currentUser.token}`,
+                }

```

Comment:
after adding the `headers.Authorization` in axiosinstance, you don't need to call this again.